LAKMobile/src/screens/LoginScreen.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { StyleSheet, Text, TextInput, View } from 'react-native';
+import LabelWrapper from '../components/LabelWrapper';
+import PrimaryButton from '../components/PrimaryButton';
+import { COLORS } from '../../constants';
+
+export default function LoginScreen() {
+    return(
+        <View style={styles.container}>
+            <LabelWrapper label='Mobile Number'>
+                <TextInput
+                style={bigInputStyle}
+                keyboardType="default"
+                />
+            </LabelWrapper>
+
+            <LabelWrapper label='4 Digit PIN'>
+                <TextInput
+                style={smallInputStyle}
+                keyboardType="numeric"
+                />
+            </LabelWrapper>
+            <PrimaryButton 
+                type='link' 
+                title='Forgot PIN?' 
+                onPress={() => console.log('Forgot PIN button pressed')}
+                style={styles.forgotPIN}
+            />
+
+            <PrimaryButton
+                type='primary'
+                title='Log in'
+                onPress={() => console.log('Login button pressed')}
+                style={styles.submitButton}
+            />
+
+            <View style={styles.signupPrompt}>
+                <Text style={styles.signupText}>Don't have an account?</Text>
+                <PrimaryButton 
+                    type='link' 
+                    title='Sign up here.' 
+                    onPress={() => console.log('Sign up button pressed')}
+                    style={styles.signupLink}
+                />
+            </View>
+        </View>
+    )
+}
+
+const styles = StyleSheet.create({
+    container: {
+        width: '100%',
+        padding: 32,
+        flex: 1,
+        alignItems: 'flex-start',
+        justifyContent: 'center',
+    },
+
+    input: {
```

Comment:
Brought this up in Aman and Assaf's pr, but since these inputs are shared this could potentially be moved outside since other pages may need it.